### ISSUE TITLE
Appveyor failing on recent PRs

### ISSUE THREAD

**Issue by grlee77** [PyWavelets/pywt#160]
I'm not sure what is going on with AppVeyor, but it is failing on the two most recent PRs made.  As far as I can tell a wheel is successfully created:
e.g.
 `C:\projects\pywt\dist\PyWavelets-0.5.0.dev0+da2774a-cp27-cp27m-win32.whl`

but pip fails to find the wheel when it attempts to install from it.

From one of the logs on #158 
```
pip install --pre --no-index --find-links dist/ PyWavelets
Ignoring indexes: https://pypi.python.org/simple
Collecting PyWavelets
  Could not find a version that satisfies the requirement PyWavelets (from versions: )
No matching distribution found for PyWavelets
Command exited with code 1
```
From what I have read, the `--pre` option we are using should be enabling development versions, so I am not sure what the problem is.

**Comment by kwohlfahrt** [comment]
I have a branch `setuptools`, that pulls out the numpy distutils in favour of normal setuptools. That means it's possible to use a much simpler AppVeyor script, which [passes](https://ci.appveyor.com/project/kwohlfahrt/pywt/build/1.0.49). However, I'm not sure this correctly caches numpy builds, so it's currently quite slow. I'll play with that a little more and then open a PR.

**Comment by kwohlfahrt** [comment]
OK, I have a branch that builds on both travis and appveyor. It uses plain setuptools and nosetests, though I'm still working on getting coverage and doctests. PR incoming.

**Comment by kwohlfahrt** [comment]
Closed by #170
